### PR TITLE
Update Python version to v3.12.4 and update some reminder

### PR DESCRIPTION
**Pull request by codenamedpkt**
We would like introducing Python v3.12.4, adds more new capabilities to this Python release. We update some reminders calling users if wanted to install `pip` and `ensurepip` by `apt`, just follow on screen instruction and get there.

### CODE REVIEW

File: packages/python/build.sh [hunk 1]
```diff
@@ -4,10 +4,11 @@ TERMUX_PKG_DESCRIPTION="Python 3 programming language intended to enable clear p
 TERMUX_PKG_LICENSE="custom"
 TERMUX_PKG_LICENSE_FILE="LICENSE"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=3.11.9
+# We get source file archive from Official Python.org's GitHub Account. Uses branch to keep the things latest.
+TERMUX_PKG_VERSION=3.12  # v3.12.4
```
Comment (Grimler91) [inline packages/python/build.sh:8]: No, omitting micro version will confuse users and not work well with services like repology

**Review by Grimler91 — COMMENTED** [review]
If you want to help then please rebase https://github.com/termux/termux-packages/pull/18078 instead, it already has several patches that probably are still needed

File: packages/python/build.sh [hunk 2]
```diff
@@ -4,10 +4,11 @@ TERMUX_PKG_DESCRIPTION="Python 3 programming language intended to enable clear p
...
 TERMUX_PKG_LICENSE_FILE="LICENSE"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=3.11.9
+# We get source file archive from Official Python.org's GitHub Account. Uses branch to keep the things latest.
+TERMUX_PKG_VERSION=3.12.4
 TERMUX_PKG_REVISION=2
-TERMUX_PKG_SRCURL=https://www.python.org/ftp/python/${TERMUX_PKG_VERSION}/Python-${TERMUX_PKG_VERSION}.tar.xz
-TERMUX_PKG_SHA256=9b1e896523fc510691126c864406d9360a3d1e986acbda59cda57b5abda45b87
+TERMUX_PKG_SRCURL=https://github.com/python/cpython/archive/${TERMUX_PKG_VERSION}/cpython-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=d833211ef08babdeecdba6447f371766f802b56731b34e0188949fb2fdeee968
```
[the remark this answers is not in the document]
Comment (codenamedpkt) [reply packages/python/build.sh:11]: They are same from python.org hosted. Sorry the correct checksum now in version v3.12.4 is 9fe6e3672a17a4d9c9ff95aed286115cf196c53eade7d7d864c05bdce0a04f2d.

File: packages/python/build.sh [hunk 3]
```diff
@@ -115,7 +116,7 @@ termux_step_create_debscripts() {
 		echo
 		echo "== Note: pip is now separate from python =="
 		echo "To install, enter the following command:"
-		echo "   pkg install python-pip"
+		echo "   pkg install python-pip. For apt, apt install python --install-recommends -yfm --autoremove --purge."
```
[the remark this answers is not in the document]
Comment (codenamedpkt) [reply packages/python/build.sh:119]: Oh I see.

**Comment by Biswa96** [comment]
There is a similar pending pull request to update python #18078 which is not as easy as this one.

**Comment by codenamedpkt** [comment]
Hi there I updated infos in #18078.

**Pull request closed without merging**